Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -38,6 +38,7 @@ public class KryptonButton : VisualSimpleBase, IButtonControl, IContentValues
         private readonly PaletteTripleOverride _overridePressed;
         private IKryptonCommand _command;
         private bool _useAsDialogButton, _isDefault, _useMnemonic, _wasEnabled, _useAsUACElevationButton;
+        private float _cornerRoundingRadius;
```

Comment:
Why is this here.. A Button already has rounding on it

Reply:
Testing, part of #596, really don't need it as the value can be fed to `CornerRounding`